packages/node_modules/@webex/plugin-meetings/src/roap/handler.js

```diff
@@ -192,8 +192,10 @@ export default class RoapHandler extends StatelessWebexPlugin {
   handleAction(session, action, meeting, correlationId) {
     let signal;
 
+
     switch (action.type) {
       case ROAP.RECEIVE_ROAP_MSG:
+        LoggerProxy.logger.log(`Roap:handler#handleAction --> RECEIVE_ROAP_MSG event captured, reciving a roap message : ${action}`);
```

Comment:
I think having `${action}` in the log like this will result in just having "[object Object]" in the output log, won't it? I think we need to use `JSON.stringify()` (same for the other log in this file)